### DISCUSSION TITLE
关于新版 wiki 中斜体统一改成粗体的一些疑问

### DISCUSSION THREAD

**Discussion by zifeiyu666** [reactjs/zh-hans.react.dev#1127] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/23133123/230836396-3988be83-00d7-4071-8a6e-5283d9ea1e10.png)
像这种情况，斜体是存在于粗体内部的，要如何处理？

**Comment by awxiaoxian2020** [comment, marked as answer]
> ![image](https://user-images.githubusercontent.com/23133123/230836396-3988be83-00d7-4071-8a6e-5283d9ea1e10.png)
> 像这种情况，斜体是存在于粗体内部的，要如何处理？

我们会在review时统一处理，先暂时这样提交pull request
